testsuites/CBLTester/CBL_Functional_tests/APITests_sc/test_basic_sc.py

```diff
@@ -0,0 +1,44 @@
+import random
+import pytest
+from keywords.utils import random_string
+
+
+@pytest.mark.usefixtures("class_init")
+class TestBasicSC(object):
+
+    def test_create_scope(self, scope_name):
+        ''' Create scope with different name '''
+        self.server_obj.create_scope(scope_name)
+
+    def test_create_collection(self, scope_name, collection_name):
+        '''
+        Test create collection
+        '''
+        self.server_obj.create_collection()
+
+    def test_create_scope_already_exist(self, scope_name):
+        '''
+        @summary: Testing count method of Dictionary API
+        '''
+        self.server_obj.create_scope(scope_name)
+        try:
+            self.server_obj.create_scope(scope_name)
+        except Exception as e:
+            if "Fail" in str(e):
+                print("Test failed as expected")
+                pass
+            else:
+                raise("Server could create another scope with same name")
+        
+    def test_remove_scope(self, scope_name):
```

Comment:
Does this test depend on one of the tests before? If it is, it might be worth to make it independent, because dependencies between tests can potentially cause problems.

Reply:
This test also independent test.  It will remove scope in database